doc/README.md

```diff
@@ -114,3 +119,75 @@ pear
 watermelon
 :   green fruit with red flesh
 ```
+
+#### Examples
+
+To define a referenceable figure use the following fencing:
+
+```markdown
+:::{.example #an-attribute-set-example}
+# An attribute set example
+
+You can add text before
+
+    ```nix
+    { a = 1; b = 2;}
+    ```
```

Comment:
````suggestion
 ```nix
{ a = 1; b = 2;}
```
````

Reply:
This breaks the rendering on Github: https://github.com/NixOS/nixpkgs/pull/264450/commits/c9e40048554fd59e0a3a5af8939dfd85a8a03744

Reply:
Ah you need four (strictly more than three) backticks in the surrounding code block!